types/src/front/lib/core_api.ts

```diff
@@ -22,6 +22,13 @@ import { LoggerInterface } from "../../shared/logger";
 
 const { CORE_API = "http://127.0.0.1:3001" } = process.env;
 
+export const EMBEDDING_CONFIG = {
```

Comment:
This is not really the best place to put it since it is not `core` specific but `front` specific.

Let's at least rename it `FRONT_DEFAULT_EMBEDDING_CONFIG`

Reply:
I agree it's front specific, I thought that since it's in types/front it would be fine :sweat_smile: Ok to put it anywhere else though